l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -216,6 +252,45 @@ def get_taxes(self):
             ('company_id', '=', self.company_id.id),
         ])
 
+    @api.multi
+    def get_ob_totals(self):
+        '''return totals of opening balance'''
+        self.env.cr.execute(
+            'select sum(l.credit), sum(l.debit), count(distinct a.id) '
+            'from account_move_line l, account_account a, '
+            '     account_account_type t '
+            'where a.user_type_id = t.id '
+            'and l.account_id = a.id '
+            'and l.date < %s '
+            'and (l.company_id=%s or l.company_id is null) '
+            'and t.include_initial_balance = true ',
+            (self.date_start, self.company_id.id, ))
+        row = self.env.cr.fetchall()[0]
+        return dict(
+            credit=round(row[0] or 0.0, 2),
+            debit=round(row[1] or 0.0, 2),
+            count=row[2] or 0
+        )
+
+    @api.multi
+    def get_ob_lines(self):
+        '''return opening balance entries'''
+        self.env.cr.execute(
+            'select a.id, a.code, sum(l.balance) '
+            'from account_move_line l, account_account a, '
+            '     account_account_type t '
+            'where a.user_type_id = t.id '
+            'and a.id = l.account_id and l.date < %s '
+            'and (l.company_id=%s or l.company_id is null) '
+            'and t.include_initial_balance = true '
+            'group by a.id, a.code',
+            (self.date_start, self.company_id.id, ))
+        for result in self.env.cr.fetchall():
+            yield dict(
+                account_id=result[0],
+                account_code=result[1],
+                balance=result[2])
```

Comment:
Please `round` this `result[2]` here to prevent errors like `Element '{http://www.auditfiles.nl/XAF/3.2}amnt': [facet 'fractionDigits'] The value '0.300000000001' has more fractional digits than are allowed ('2').`